lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -223,3 +225,31 @@ def get_blocks_and_check_against_expected(self, user, expected_blocks):
             self.transformers,
         )
         assert set(block_structure.get_block_keys()) == set(self.get_block_key_set(self.blocks, *expected_blocks))
+
+    @override_waffle_flag(EXAMS_IDA, active=False)
+    @patch('lms.djangoapps.course_api.blocks.transformers.milestones.get_attempt_status_summary')
+    def test_exams_ida_flag_off(self, mock_get_attempt_status_summary):
```

Comment:
I'm just testing the waffle flag based on whether there is a call to proctoring.